scripts/SasNetworkGuard.psm1

```
@@ -0,0 +1,184 @@
+}
+
+function Get-SasLocalNetworkText {
+    [CmdletBinding()]
+    param()
+    if ($env:SAS_NETWORK_GUARD_IPCONFIG_FIXTURE) {
+        return (Get-Content -LiteralPath $env:SAS_NETWORK_GUARD_IPCONFIG_FIXTURE -Raw)
+    }
+    try { return (& ipconfig /all 2>$null | Out-String) } catch { return '' }
+}
+
+function Test-SasIpInCidr {
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Test-SasIpInCidr' does not have a help comment.

(PSProvideCommentHelp)